### DISCUSSION TITLE
How do I upgrade request/responses to use Http2

### DISCUSSION THREAD

**Discussion by StevenBendell** [remix-run/remix#4642] [Q&A]
I'm trying to implement server side events using the library https://www.npmjs.com/package/@wehriam/remix-sse from the discussion https://github.com/remix-run/remix/discussions/2622. SSE has a limit of 6 connections when not used over Http2 which I want to exceed and therefore will want to use Http2 for. 
I couldn't find a way to do this in remix that anyone has documented anywhere

**Comment by machour** [comment]
Hey, I think [http2-express-bridge](https://www.npmjs.com/package/http2-express-bridge) can come handy here, if you're using the express adapter.
